rabbit-escape-engine/src/rabbitescape/engine/BehaviourTools.java

```diff
@@ -63,6 +63,11 @@ public boolean rabbitIsFalling()
         }
     }
 
+    public boolean rabbitIsClimbing()
+    {
+        return rabbit.state.toString().contains( "CLIMB" );
+    }
+    
```

Comment:
No, and I am suspicious of inheritance as a solution for any problem so would want to explore other options and/or see a prototype.